src/Benda95280/MyEntities/MyEntities.php

```diff
@@ -198,31 +231,83 @@ public static function getPlayerCustomItem(string $name, string $nameFinal, arra
+                new ByteArrayTag("GeometryData", $properties->skin->getGeometryData())
+            ]),
+            self::arrayToCompTag((array)$properties, "MyEntities"),
+        ]);
+        if ($properties->usable && $properties->usable["skinchange"] == 1) {
+            $compoundTag->setTag(new ByteArrayTag('skin_empty', MyEntities::createSkin($properties->skin->getSkinId() . "_empty")));
+        }
+        return $item->setCustomBlockData($compoundTag)
+            ->setCustomName(TextFormat::colorize('&r' . $properties->name, '&'));
+    }
+
+    /**
+     * @param CustomEntityProperties $properties
+     * @return Item
+     * @throws \InvalidArgumentException
+     * @throws \RuntimeException
+     */
+    public static function getPlayerCustomItemVehicle(VehicleProperties $properties): Item
+    {
+        $item = (ItemFactory::get(Item::BOOKSHELF));
+        $compoundTag = new CompoundTag("", [
+            new CompoundTag("Skin", [
+                new StringTag("Name", $properties->skin->getSkinId()),
+                new ByteArrayTag("Data", $properties->skin->getSkinData()),
+                new ByteArrayTag("CapeData", ""),
+                new StringTag("GeometryName", $properties->skin->getGeometryName()),
+                new ByteArrayTag("GeometryData", $properties->skin->getGeometryData())
+            ]),
+            self::arrayToCompTag((array)$properties, "MyEntities"),
+        ]);
+        if ($properties->usable && $properties->usable["skinchange"] == 1) {
+            $compoundTag->setTag(new ByteArrayTag('skin_empty', MyEntities::createSkin($properties->skin->getSkinId() . "_empty")));
+        }
+        return $item->setCustomBlockData($compoundTag)
+            ->setCustomName(TextFormat::colorize('&r' . $properties->name, '&'));
+    }
+
+    private function initializeLanguage()
+    {
+        if (!isset(self::$miscList['language'])) {
+            self::logMessage("Language not set, English applied", 0);
             self::$miscList['language'] = 'en';
-			//TODO: Error here if language is missing:
-			//[Server thread/CRITICAL]: ErrorException: "Undefined index: language" (EXCEPTION) in "plugins/MyEntities/src/Benda95280/MyEntities/MyEntities" at line 205
-		}
-		$languageSet = self::$miscList['language'];
-		//Get all language file
-		$language = [];
-		foreach($this->getResources() as $resource){
-			if($resource->isFile() and substr(($filename = $resource->getFilename()), 0, 5) === "lang_"){
-				$language[substr($filename, 5, -4)] = yaml_parse(file_get_contents($resource->getPathname()));
-			}
-		}
-		//Check if language exist, has set in config
-		if (isset($language[$languageSet])) {
-			self::$language = $language[$languageSet];
-		}
-		else {
-			self::$language = $language["en"];
-			self::logMessage(sprintf(self::$language['init_lang_notexist'], $languageSet), 0);
-		}
-	}
+            //TODO: Error here if language is missing:
```

Comment:
Wasn't fixed in the past ?

Reply:
Haven't yet pulled an update from master i guess